packages/bitcore-node/src/routes/api/wallet.ts

```diff
@@ -188,7 +188,7 @@ router.post('/:pubKey', authenticate, async (req: AuthenticatedRequest, res) =>
 router.get('/:pubKey/transactions', authenticate, async (req: AuthenticatedRequest, res) => {
   let { chain, network } = req.params;
   try {
-    return ChainStateProvider.streamWalletTransactions({
+    return await ChainStateProvider.streamWalletTransactions({
```

Comment:
if this is streaming why does it await?

Reply:
So interesting thing I learned. If there's a throw inside of an async method that streams, then you'll get an unhandled promise rejection if you don't await that method.

Reply:
Error

Reply:
eRroR